backend/app/utils/jsonl_formatter.py

```
@@ -0,0 +1,354 @@
+                        continue
+                
+                output_dict[mapping.target_field] = source_value
+        
+        # 添加属性字段
+        for attr_key, attr_value in block.attributes.items():
+            if attr_key not in output_dict:  # 避免覆盖已映射的字段
+                output_dict[attr_key] = attr_value
+        
+        # 排除空字段
+        if self.config.exclude_empty_fields:
+            output_dict = {k: v for k, v in output_dict.items() 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)